app/controllers/admin/projects_controller.rb

```diff
@@ -44,7 +47,15 @@ def update
   end
 
   def move_project_to_channel
-    MoveProjectToChannel.new(params[:project_id],params[:channel_id]).call
+    project = Project.find_by(id: params[:project_id])
```

Comment:
And also, you might create another error for the case when the service returns false. If it happens, the problem is because the project is not able to be moved to another channel.